README.md

```diff
@@ -74,81 +82,83 @@ As we only support CPX library now, other libraries (i.e. simpleio) can’t run
+#### I. Take a look at the "Device Simulator Express: Getting Started" Command.
+1. Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL + SHIFT + P` / `CMD + SHIFT + P` to open the command palette).  
+<img alt="Deploy to Device" src='assets/readmeFiles/getting_started.png'>
+1. Choose the `CPX` option from the dropdown.
+2. Read, copy and learn some of the things you can do with the simulator!
 
-1. Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL+SHIFT+P` to open the command palette).  
-   <img alt='"New File" animation' src=https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/newFile.gif>
-2. Name and save your file somewhere, and we’re good to go! (3)
-3. Start with some examples: you can find examples files and tutorials inside the comments, as well as in the notification pop up when you run the `"Device Simulator Express: [Circuit Playground Express] New File"` Command.
+#### II. Start with the "Device Simulator Express [Circuit Playground Express]: New File" Command.
 
-<img alt='How to find example code screenshot' src=https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/findExamples.jpg>
+1. Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL + SHIFT + P` / `CMD + SHIFT + P` to open the command palette).  
+   <img alt='"New File" animation' src='assets/readmeFiles/cpx/cpx-new-file.gif'>
+2. Name and save your file somewhere, and we’re good to go!
+3. Start with some examples: you can find examples files and tutorials inside the comments, as well as in the notification pop up when you run the `"Device Simulator Express: [Circuit Playground Express] New File"` Command.
 
-#### 2. Start from an existing python file.
+#### III. Start from an existing python file.
 
 1. Open the folder or your .py file in Visual Studio Code.
 2. Run `Device Simulator Express: [Circuit Playground Express] Open Simulator` from the command palette or icon in the editor toolbar.
 
-#### 3. Run your code on the simulator.
+#### IV. Run your code on the simulator.
 
-<img alt='How to run the simulator animation' src='https://github.com/microsoft/vscode-python-devicesimulator/blob/dev/assets/readmeFiles/run.gif?raw=true'>
+<img alt='How to run the simulator animation' src='assets/readmeFiles/cpx/cpx-run.gif'>
 
-- Run `Run Simulator` from the command palette or icon in the editor toolbar.
-- You can use the `Play` or `Refresh` button on the simulator webview.
+1. Run `Run Simulator` from the command palette or use the `Play` button on the simulator webview.
 
-#### 4. Deploy your code to the physical device
+#### V. Deploy your code to the physical device
 
 Before deploying the python code to your CPX device, you need to format your device following these tutorials:

```

Comment:
".., you need to format your device **by** following these tutorials"

Reply:
fixed